pkg/packagekit/package_fpm.go

```diff
@@ -0,0 +1,147 @@
+package packagekit
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+)
+
+type outputType string
+
+const (
+	Deb outputType = "deb"
```

Comment:
would you consider taking advantage of the `PackageFlavor` enum instead of redefining these guys?

Reply:
TBH I'm not really happy with any of that. Talking aloud:

PackageFlavor is part of `packaging`, not `packagekit` 

PackageFlavor is (probably) broader than what fpm can handle. So re-using it may lead to fpm errors down the road. 

This is a mandatory thing, which might fit better as a positional argument, and not as functional parameter.